routes.py

```diff
@@ -0,0 +1,149 @@
+from flask import render_template, abort, request, redirect, url_for, flash
+from rwrs import app
+import models
+import rwr
+
+
+@app.route('/')
+def home():
+    online_players_data = models.ServerPlayerCount.server_players_data()
+    servers_online_data = models.ServerPlayerCount.servers_data()
+    servers_active_data = models.ServerPlayerCount.servers_data(active_only=True)
+
+    servers_data = [
+        servers_online_data,
+        servers_active_data
+    ]
+
+    return render_template(
+        'home.html',
+        online_players_data=online_players_data,
+        servers_data=servers_data
+    )
+
+
+@app.route('/my-friends')
+def my_friends():
+    return render_template(
+        'manage_friends.html'
+    )
+
+
+@app.route('/players')
+@app.route('/players/<username>')
+def player_stats(username=None):
+    if not username:
+        username = request.args.get('username').strip()
+
+        # Redirect to a SEO-friendly URL if the username query parameter is detected
+        return redirect(url_for('player_stats', username=username), code=301)
+
+    if not username:
+        abort(404)
+
+    scraper = rwr.DataScraper()
+
+    player = scraper.search_player(username)
+
+    if not player:
+        flash('Sorry, the player "{}" wasn\'t found. Maybe this player hasn\'t already played on a ranked server yet.'.format(username), 'error')
```

Comment:
Create a constant for this message, and also mention if this player plays for the first time he have to wait tomorrow for the stats to be available